src/main/kotlin/com/linker/linkerapi/rental/entity/Rental.kt

```diff
@@ -0,0 +1,14 @@
+package com.linker.linkerapi.rental.entity
+
+import com.linker.linkerapi.common.entity.BaseEntity
+import jakarta.persistence.Entity
+
+@Entity
+class Rental(
+    val equipmentId: Long,
```

Comment:
이렇게 해도 되지만 spring boot에서는 relation으로도 표현합니다.
```suggestion
   @ManyToOne
    val equipment: Equipment,
```

이렇게 작성하면 되는데요 왜 ManyToOne을 사용해야 되는지는 한 번 찾아보시고 여기다가 이유를 답글로 달아주세요!

Reply:
우오,, 되게 중요한 어노테이션이었네요.. 찾아보면서 많이 배웠습니다 ,, 한 기자재가 여러 대여 신청이랑 연결될 수 있고 (즉, 여러 Rental 인스턴스가 하나의 Equipment를 참조할 수 있으니까) ManyToOne으로 표현할 수 있습니다 ! 그리고 데이터베이스에선 외래키 사용해서 관계를 관리하는데 JPA는 이걸 통해서 equipment 데이터를 가져올 수 있습니다 ! 어.. 좀 뭔가 중구난방으로 설명한거 같은데 하여튼 엔티티의 관계를 객체지향적으로 표현하기 위함이라고 볼 수 있네요